expression/builtin_miscellaneous_vec_test.go

```diff
@@ -80,6 +80,7 @@ var vecBuiltinMiscellaneousCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETDecimal, childrenTypes: []types.EvalType{types.ETString, types.ETDecimal}},
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETInt}},
 		{retEvalType: types.ETReal, childrenTypes: []types.EvalType{types.ETString, types.ETReal}},
+		{retEvalType: types.ETJSON, childrenTypes: []types.EvalType{types.ETString, types.ETJSON}},
```

Comment:
`types.ETJSON` -> `types.ETJson`.

Reply:
@Baytwt Friendly ping, please fix this problem and sign our CLA license.

Reply:
fixed